sdk/metric/metricdata/data.go

```diff
@@ -211,6 +213,14 @@ type Extrema[N int64 | float64] struct {
 	valid bool
 }
 
+// MarshalJson converts the Extrema value to JSON number
+func (e *Extrema[N]) MarshalJSON() ([]byte, error) {
+	if !e.valid {
+		return []byte("0"), nil
+	}
+	return []byte(json.Number(fmt.Sprint(e.value))), nil
```

Comment:
Wy not simply?

```suggestion
	return json.Marshal(e.value)
```

Reply:
Well, not really. I've started with implementation of MarshalText, then refactored it to MarshalJSON so I wasn't thinking on replacing the whole statement. I'm waiting on resolution of [discussion](https://github.com/open-telemetry/opentelemetry-go/pull/4827#discussion_r1453182773) regarding MarshalText, so I'll hold back with simplifying that one till it's resolved.

Reply:
It is resolved. You can address this and other comments.